examples/case_studies/bayesian_workflow.myst.md

```diff
@@ -0,0 +1,1121 @@
+
+The process works as follows:
+
+1. Pick a point from the prior $\theta_i$
+2. Generate data set $x_i \sim f(\theta_i)$ where $f$ is our likelihood function (e.g. normal)
+3. Rinse and repeat $n$ times
+
+```{code-cell} ipython3
+with model_exp1:
+    prior_pred = pm.sample_prior_predictive()
+```
+
+```{code-cell} ipython3
+prior_obs = prior_pred.prior_predictive["obs"].values.squeeze().T
+
+fig = go.Figure()
+for i in range(min(100, prior_obs.shape[1])):  # Show max 100 traces
+    fig.add_trace(
+        go.Scatter(
+            x=list(range(len(prior_obs[:, i]))),
+            y=prior_obs[:, i],
+            mode="lines",
+            line=dict(color="steelblue", width=0.5),
+            opacity=0.4,
+            showlegend=False,
+        )
+    )
+
+fig.update_layout(
+    title="Prior predictive",
+    xaxis_title="Days since 100 cases",
+    yaxis_title="Positive cases",
+    yaxis=dict(range=[-1000, 1000]),
+    xaxis=dict(range=[0, 10]),
+    template="plotly_white",
+)
+```
+
+:::{admonition} What's Wrong With This Model?
+:class: warning
+
+Our prior predictive check reveals several problems:
+1. **Negative cases**: Impossible in reality
+2. **Starting near zero**: We conditioned on 100+ cases
+3. **Normal likelihood**: Allows negative values and symmetric errors
+
+This demonstrates the value of prior predictive checking!
+:::
+
+
+
+Let's improve our model. The presence of negative cases is due to us using a Normal likelihood. Instead, let's use a `NegativeBinomial`, which is similar to `Poisson` which is commonly used for count-data but has an extra dispersion parameter that allows more flexiblity in modeling the variance of the data.
+
+We will also change the prior of the intercept to be centered at 100 and tighten the prior of the slope.
+
+The negative binomial distribution uses an overdispersion parameter, which we will describe using a gamma distribution. A companion package called `preliz`, a library for prior distribution elicitation, has a nice utility called `maxent` that will help us parameterize this prior, as the gamma distribution is not as intuitive to work with as the normal distribution.
+
+```{code-cell} ipython3
+dist = pz.Gamma()
+
+pz.maxent(dist, lower=0.1, upper=20, mass=0.95);
+```
+
+```{code-cell} ipython3
+px.histogram(x=dist.rvs(1000), nbins=20, title="Gamma Distribution Samples")
+```
+
+```{code-cell} ipython3
+t = df_country.select("days_since_100").to_numpy().flatten()
+confirmed = df_country.select("confirmed").to_numpy().flatten()
+
+with pm.Model() as model_exp2:
+    # Intercept
+    a = pm.Normal("a", mu=100, sigma=25)
+
+    # Slope
+    b = pm.Normal("b", mu=0.3, sigma=0.1)
+
+    # Exponential regression
+    growth = a * (1 + b) ** t
+
+    alpha = alpha = dist.to_pymc("alpha")
+
+    # Likelihood
+    pm.NegativeBinomial("obs", growth, alpha=alpha, observed=confirmed)
+```
+
+```{code-cell} ipython3
```

Comment:
https://pymcio--797.org.readthedocs.build/projects/examples/en/797/case_studies/bayesian_workflow.html#step-3-run-prior-predictive-check I think this cell should be generating a plot at the bottom of the linled section but I only see the prior sampling logs as output

Reply:
Thank you.